core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -194,79 +196,133 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
+                   ng-click="$event.stopPropagation()"
+                   ng-if="exploration.status !=='private' && activeExplorationId === exploration.id">
+            <div class="stat-block">
+              <div class="heading-text"
+                   ng-click="toggleSubDropdown(EXPLORATION_DROPDOWN_STATS.OPEN_FEEDBACK, $event)">
+                <[explorationStats[exploration.id] ? explorationStats[exploration.id].open_feedback.length + ' Open Feedback': 'Retrieving Feedback Data ...']>
+                <span class="fa"
+                      ng-if="explorationStats[exploration.id].open_feedback.length !==0 && !checkMobileView()"
+                      ng-class="activeSubDropdown === EXPLORATION_DROPDOWN_STATS.OPEN_FEEDBACK ? 'fa-caret-up': 'fa-caret-down'">
+                </span>
               </div>
-              <ul ng-if="exploration.status !== 'private'"
-                  layout="row"
-                  layout-wrap
-                  class="metrics"
-                  layout-align="space-between center">
-                <li flex="50">
-                  <span class="protractor-test-exp-summary-tile-rating">
-                    <span class="fa fa-star fa-lg"
-                          tooltip="<['I18N_LIBRARY_RATINGS_TOOLTIP' | translate]>"
-                          tooltip-placement="top">
-                    </span>
-                    <span>
-                      <[(getAverageRating(exploration.ratings) | number:1) || 'N/A']>
-                    </span>
-                  </span>
-                </li>
-
-                <li flex="50">
-                  <span class="fa fa-eye fa-lg"
-                        tooltip="<['I18N_LIBRARY_VIEWS_TOOLTIP' | translate]>"
-                        tooltip-placement="top">
-                  </span>
-                  <[exploration.num_views]>
-                </li>
-
-                <li flex="50">
-                  <span class="fa fa-clock-o fa-lg"
-                        tooltip="<['I18N_LIBRARY_LAST_UPDATED' | translate]>"
+              <div class="stat-block-values"
+                   ng-if="explorationStats[exploration.id].open_feedback.length !==0"
+                   ng-show="activeSubDropdown === EXPLORATION_DROPDOWN_STATS.OPEN_FEEDBACK || checkMobileView()">
+                <div ng-repeat="feedback in explorationStats[exploration.id].open_feedback">
+                  <[feedback.text | truncate: 100]> (by <a ng-href="/create/<[exploration.id]>#/feedback"><[feedback.author_username || 'a learner']></a>)
+                </div>
+              </div>
+            </div>
+            <div class="stat-block" ng-if="!checkMobileView()">
+              <div class="heading-text"
+                   ng-click="toggleSubDropdown(EXPLORATION_DROPDOWN_STATS.TOP_UNRESOLVED_ANSWERS, $event)">
+                <[explorationStats[exploration.id] ? exploration.top_unresolved_answers.length + ' Unresolved Answers from users': 'Retrieving Answers Data ...']>
```

Comment:
Similar to above for unresolved answers, you should use the total number of unresolved answers, and not the length of the data.